tutorials/feature-use/global_modeling.ipynb

```diff
@@ -0,0 +1,1084 @@
41+c0rhQMxnF2tl6oK4z4qBYOU0sbNTrPEj2EJDoBknLBvYrLmH7K8RCnp8LrtUdFOo16qt+0DXogB86f4B8zWRmATGmEmWy/dcdk67xDxCLh+JJ2HJ5FJ9+09RvNqK7HaisH7jv8Fthc940LVz7sIqEc08sMWVwirLljloT7bNQ7In9sLuQczskrtMpK8rx4rUiS0kVWw+M2bAStVJmmDeWS1VDTcRiMbRq1QoNGzZk23X6C9p5z7GLhy7A3LlzUVjIlDgME/0v/SEev/f2fwv8FRcXaxrznIRC+z6SeyJlwY98tt27j20b1RsE1ROX/YkpycxxF2wvvfQS3nzzTckUhbjGfHFA0m4UlmXhlVdewahRowIXAZQBV6u8q1evduVOSaYvyD34PxVpgPwtRCQSwZnndscP/rkMMztXI+f0TPe9vt+7QAZChuE5WDkds3ThEQfZpuYOabI0IPKbHiC89pzzxM3AyoGJRBxYPhWL1k7FxJrrUxLlVf7V1KlT3c97+sZyBuqULFHOCpkG0HkQ2jZrhC3fGMDKd9yJ0FayEhndh6OmY7TOxpdrr70Wb731FuLxOEKhEC648CLguSWoWfo6bA6O23SWTEeow/m9NSuBZe/ImQ2pcz7hZUBcFQsbU197GbGbbkBe5x6+0uz8sf0lEPDTKy5BVdVm8TAloFlZigd+ezcG9e+NSCSCAwcOuONGvC4aPwwc49+3NbPkcW7QCIhXi9dCYTRqf57fkEUn3q9OwjrjBw4ceMbRNMWEULYUKYFkqw5CqYIad0oXy5MyNd/RwomDTLKpjb0qG0uoYCYzS2T/VaoCqXGQ0QVRR9RFA9dmNgx99aBVRyE75yplZMgghLLVhw4IYLnkDbF/tCigJjy1NA4ITuehfaKBT3fNdx4kzjsfR1oUsMxPymjf1wNXtA8EtGlxQWCZKw00z/NXcFRuNIUOmOpeo21zabN1c0WTkzr2rTp64KpxM/k99W++f4FW0tBndkmJZdMngqIS8OxJWrbT1LTde7Fpjvj8oJsZwBf38LrfRHHbOyWYv6lKuz0XfRsm0GOkV7mQFiW2N050flQqg0rBoPeoaVTVzCbO+Pp5vmtr06dLEaponKIR2FkYhzK8BlNeudHRjNRKTV4Pj8a2dZUAyIBEO/nU97uWsli3YcdrUFhYiN/85jdIZvf3fSMajfqribaNF154wVVZ4I3lOs19CrU5rlOnTtiwYYOYvyxL3MPOvEpzbjQadUCkmMd7Dr38386M0jZ9zfhUXVDOWWVlJQ6tVmkhNmpqa5V7QZzXZDKJzz//3Pe7TZo0QatWrSQhgSBBgOnTp2PWrFnaJr1IJIIf/vCHPvm5yspKH/30uus01Z7/YKQB8rcQsVgMv33uLaypaY0XurfG74ac7b53Xu++3s3Ks1Cr3wWQlG72SCSCs/7vMWwtWSZKbFJjiHeTkOybZdv6knnZUuwCcNNNYhLUgeRYLIbNmzcHHlM0GnU0ltkNYyUx69lHMLFLa9z/i18A1TXYQmVnx5oz1LARktXfSPtds7cC6Agvu1a1zfd7hmnitNNOc8fDMAysX7cWePs+JNXsX8UGlrEWHc4A8NhtPwJqa33yQm52TpI18iLp6Fz+9Nzuir1sHC+/9DLwTQd3gso9tzuip+f5NJnjca8S8P7770vbdx86ti24YSqoapnvTcRmyGmws10Fh0Z5XZwFS0D5FtC4bWmoEIBfoom+S/8mowJazPGSPAHJUEjsi2H4s5FmSEzeO9Z7x8iBK5c527zSyzSpltFK5cLdx65DxWs6+Ta6Nrg2s04OiVt9E++Yg8jojd75KFvCwKTtqTH0vcrvXrh+npO1cRYMe7bIvFVV9o60oHk2UJdFbnU2sKvUvcdgmMLtb8sKWd84ERdjx3WodcDxst95EnF8oaKGDpgGgen18xRzibgAapxeZoblzGOf0R5X0gyJv3XB+woaZ8sKNOEMfWZ3zSxP5WPFVMHxdraTwbJkgzs2RyxoP3aXeQDfEue3c6vfoEEooOGYA1ui7ahW0xS27bkzck42UZWo6kI6xV8ud1wQHfBWvtZpmhzhbZNXktgiv0P3fujfrpmrmS81mtK9ECSLp6pqqGoaC6c4PQzsGDlvN/ZqMHUrIDZt2oSbbroJF4z5PXBqH83Q+cfTx3mlsTi4RwLIrVq1cv9NqlGWZSGZTGLDFxu9bRuGxKumJIynriEWOWWfLQUQLLtXH4pBJBLB+eefj4ULF8pv5PX09MOdMQyFQuIYNFUmC6ZX0XL7J4Jj//79+Oqrr1J+hgcpX+jcdF9//XXf5/ft25eSfnoyRBogn+BQrSeXnVENDPm9+75FXSAH93juZdvWuA/eeFzWaHRv0Jb5cvmYRTweR0lJCVp07OazdW2c1QRf7/eyG5MnT3YBMreUvP322322n19+6T0QIpEIwgVDkfh0pvSZRLwWTzzxBGqrq+HsMOPv2Qo4dmLrp84xB5W7BI8LAOKOtEwikcCM6e/osx4EJB0Zo8LCQvHgijvqBFxeSAUNgEy7MAyEQiFs27YNL78wWV5s2DZemvKCmKgcILO9Ty4QaStnMQxD4iAfOnRIc4xO7PxCTyegbB/XJXUyNTZYZza/HjavEoYnjZsGGDEoJXyVkkHB/82BavVBkf3i9tE7PnfUM5ws2A4v+wBANN5Qc5466aoGEGVLBCDn++azzAVc+1vOU6ZzqRo8lJcEUBQMoWLS+my/WYgKClwtWgo2aXJwQA5nxMlUrWGTygJFopJYohJC1uc8WrZnusUQQG3wLf7zIekbO/tZWSr+4xUGhefpujcGAWOKbsNEmZ6AphlCxhlt9Z/VuZlVbhLXbH4/7xol4ETjfvXD9dMapv0l8EdxRkf951fN8P/tHG//PI/idf+wc/B//c5CW6DO/fhRT6EOEE8GVFh4JcTXMKsJ3gimLq5Mh4IBME67wuNePw/Ab/zbVRbB7br2BA5tgZspNsP+THGqoHuNTIbo
```

Comment:
"Please note that in the case of local normalization the test metrics are not re-scaled but displayed in normalized scale."

Reply:
@MateusGheorghe this might still be missing?

Reply:
Done